rust/index/src/spann/types.rs

```diff
@@ -94,6 +97,10 @@ const NUM_CENTROIDS_TO_SEARCH: u32 = 64;
 const RNG_FACTOR: f32 = 1.0;
 #[allow(dead_code)]
 const SPLIT_THRESHOLD: usize = 100;
+const NUM_SAMPLES_FOR_KMEANS: usize = 1000;
+const INITIAL_LAMBDA: f32 = 100.0;
+const REASSIGN_NBR_COUNT: usize = 8;
```

Comment:
more comments plz

Reply:
cutout a task in tracker to add comments across the entire spann codebase